docker/generic/start_proxy.py

```diff
@@ -189,6 +189,11 @@ def make_argparser():
        backward compatible for ESPv1 and will be deprecated.
        Please use the flag --listener_port.''')
 
+    parser.add_argument('-S', '--ssl_port', default=None, type=int, help='''
+    Expose a port for HTTPS requests.  Accepts both HTTP/1.x and HTTP/2
+    secure connections. Requires the certificate and key files
+    /etc/endpoints/ssl/envoy.crt and /etc/endpoints/ssl/envoy.key''')
```

Comment:
Are they already built in our image?

Reply:
Not necessary, these files are from API producers.